src/pages/Editor.vue

```diff
@@ -1278,7 +1261,6 @@ export default {
     addNewItem(questionType) {
       this.player.pause();
       this.startLoading();
-      setTimeout(() => {}, 5000);
```

Comment:
why was this removed?

Reply:
Kuch use hi nhi dikha iska. It was just waiting 5 seconds